src/tpm2/VendorString.h

```diff
@@ -62,47 +62,67 @@
 #ifndef VENDORSTRING_H
 #define VENDORSTRING_H
 
-/* Define up to 4-byte values for MANUFACTURER.  This value defines the response for
-   TPM_PT_MANUFACTURER in TPM2_GetCapability(). The following line should be un-commented and a
-   vendor specific string should be provided here. */
-#define    MANUFACTURER    "IBM"
-
-/*     The following #if macro may be deleted after a proper MANUFACTURER is provided. */
-#ifndef MANUFACTURER
-#error MANUFACTURER is not provided.				 \
-    Please modify VendorString.h to provide a specific	  	 \
-    manufacturer name.
+#ifndef _STRINGIFY
+#define _STRINGIFY(x) #x
+#endif
+#ifndef STRINGIFY
+#define STRINGIFY(x) _STRINGIFY(x)
```

Comment:
These could be useful elsewhere. Move to a Utils.h?

Reply:
Found it already defined in `tpm_library_intern.h`